src/cbor/cbor.ts

```diff
@@ -0,0 +1,540 @@
+/* tslint:disable:max-classes-per-file */

```

Comment:
Can you please include source and licensing information here?

Reply:
Done. 
A bit of a correction about the reason of the port: Their latest release is missing on the npm, and adding the github repo as dependency forces different utilities such as gulp.

Reply:
Are there not any alternative CBOR libraries that could be used from NPM?

I'm a bit concerned about the amount this increases the package size by, for just one chain.

Reply:
Tried different crc implementations and replaced crc dependency with best/smallest one.

Updated filesize increase:
378286 -> 387385 (~9k)